docs/pwncat.api.html

```diff
@@ -18111,4 +18111,4 @@ <h4><code><a title="pwncat.TransformSafeword" href="#pwncat.TransformSafeword">T
 <p>Generated by <a href="https://pdoc3.github.io/pdoc"><cite>pdoc</cite> 0.9.2</a>.</p>
 </footer>
 </body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
There's the trailing newline too much. I've tried to commit this via GitHub UI, but the commit does not seem to remove it

Reply:
Yep, should be fixed now

Reply:
Looks all good to me. I will let CI figure out the rest.